helsingborg-stad/helsingborg-io-sls-api
# feat(metrics): add sqs metrics

## Explain the changes you’ve made

Added SQS metrics (approximate message count for every queue + dlq flag)

## Explain why these changes are made

Nice to have - in the future will be used to drive alarms.

## How to test

Concrete example:

1. Checkout this branch
2. Deploy `metrics-api`
3. Go to your metrics endpoint and verify sqs stats are showing
